docs/newbs_getting_started.md

```diff
@@ -66,6 +66,8 @@ You will need to install Homebrew. Follow the instructions on https://brew.sh.
 
 Install the QMK CLI by running:
 
+    brew tap osx-cross/arm
+    brew tap osx-cross/avr
```

Comment:
As far as I'm aware, this should not be needed: 
https://github.com/qmk/homebrew-qmk/blob/fc2b5be6e530f7c51268e6ae7fa6a5543d2e765f/Formula/qmk.rb#L25-L26

Reply:
Apologies if this was obvious, but why do those lines suggest that tapping the osx-cross/arm and avr casks is not needed? Seems like formula depends on them and they're not tapped by default, which is why I was getting the error on a brand new brew install.

(this can be reproduced with a new brew install if you try to install QMK)

Reply:
Just wanted to add a quick ping here - this still seems relevant, but please do let me know if I missed something

Reply:
From what I understand, having the full name there should properly tap it.  

However, I'm not an expert with brew, and may be wrong.